tests/errorHandling.test.ts

```diff
@@ -0,0 +1,135 @@
+import { expect } from 'chai';
```

Comment:
you are testing getReadableErrorMessage in this file. I would name the file: getReadableErrorMessage.test.ts

Reply:
https://github.com/Voltz-Protocol/v1-sdk/pull/134/commits/5221a1df7162e7aa3a6672a6410b3c83fbab894b